lib/internal/errors.js

```diff
@@ -110,6 +110,8 @@ E('ERR_CHILD_CLOSED_BEFORE_REPLY', 'Child closed before reply received');
 E('ERR_CONSOLE_WRITABLE_STREAM',
   'Console expects a writable stream instance for %s');
 E('ERR_CPU_USAGE', 'Unable to obtain cpu usage %s');
+E('ERR_CRYPTO_ECDH_INVALID_FORMAT',
+  (format) => `Invalid ECDH format: ${format}`);
```

Comment:
nit: use printf-like syntax?

Reply:
I can, yeah, I tend to prefer this way but it's no biggie for me :-)

Reply:
I don't mind keeping it this way if that's what you prefer 👍

Reply:
nah... I'll change it. It's no biggie and I don't feel strongly about it